visualizer/src/Project/ForceLoadOutputModal.js

```diff
@@ -0,0 +1,58 @@
+import { Button } from '@mui/material';
+import Box from '@mui/material/Box';
+import Modal from '@mui/material/Modal';
+import Typography from '@mui/material/Typography';
+import { useSelector } from '@xstate/react';
+import React from 'react';
+import { useIdb } from './ProjectContext';
+
+const style = {
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: 400,
+  bgcolor: 'background.paper',
+  border: '2px solid #000',
+  boxShadow: 24,
+  p: 4,
+};
+
+function ForceLoadOutputModal() {
+  const idb = useIdb();
+  const open = useSelector(idb, (state) => state.matches('promptForceLoadOutput'));
+
+  const id = new URLSearchParams(window.location.search).get('projectId');
+
+  return (
+    <Modal open={open} onClose={() => idb.send('USE_LOCAL_PROJECT')}>
+      <Box sx={style}>
+        <Typography variant='h6' component='h2'>
+          Confirm Force Reload from Output Bucket
+        </Typography>
+        <Typography sx={{ my: 2 }}>
+          Project <strong>{id}</strong> has been loaded previously, but you've opened a link to
```

Comment:
<img width="490" alt="Screen Shot 2022-06-17 at 2 23 03 PM" src="https://user-images.githubusercontent.com/20373588/174403519-172c697b-8b0c-4d4b-bc52-157f59d50ea3.png">
The current deployment has extra text at the beginning of the message, but I'm not sure where it is coming from.

Reply:
Fixed in 7af53fe787aad573e0f681f15f74f68db6b9e758 and in current deployment on label-dev.deepcell.org